src/index.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import App from "./components/App.js";
+const rootEl = document.getElementById('app');
```

Comment:
yup! For react you need to give it an initial dom element to hang everything off of.